src/sentry/static/sentry/app/views/settings/projectAlerts/projectAlertHeader.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import SentryTypes from 'app/sentryTypes';
+import Button from 'app/components/button';
+import ListLink from 'app/components/listLink';
+import NavTabs from 'app/components/navTabs';
+
+import Tooltip from 'app/components/tooltip';
+
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+
+import withOrganization from 'app/utils/withOrganization';
+
+export default withOrganization(
```

Comment:
Typically I've been writing this as 

```jsx
class Whatever {
   ...
}

export default withOrganization(Whatever);
```

to reduce the indentation.

Reply:
I do that too as using multiple HOCs gets gnarly with the inline form. But :bike::house: